ibm_catalog.json

```diff
@@ -65,6 +65,22 @@
               ],
               "service_name": "watsonx-orchestrate",
               "notes": "Required for creating a new instance of watsonx Orchestrate."
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
+              "service_name": "sysdig-monitor",
+              "notes": "[Optional] Required if you are consuming Observability DA which sets up Cloud monitoring."
+            },
+            {
+              "role_crns": [
+                "crn:v1:bluemix:public:iam::::serviceRole:Manager",
+                "crn:v1:bluemix:public:iam::::role:Editor"
+              ],
+              "service_name": "logs",
+              "notes": "[Optional] Required if you are consuming Observability DA which sets up Cloud logs."
```

Comment:
DA -> deployable architecture

Reply:
We should add permissions for COS and KMS too - that is required for Observability